layouts/partials/banner.md

```diff
@@ -1,17 +1,28 @@
 {{/* cSpell:ignore contribfest */ -}}
 {{ if .Params.show_banner -}}
+  {{ $currentDate := time.Now | time.Format "2006-01-02" }}
+  {{ $sortedAndFiltered := slice }}
+
+  {{/* Sort entries by endDate and filter out past ones */}}
+  {{ $sorted := sort .Site.Data.banners "endDate" }}
+  {{ range $sorted }}
+    {{ if ge .endDate $currentDate }}
+      {{ $sortedAndFiltered = $sortedAndFiltered | append . }}
+    {{ end }}
+  {{ end }}
+
+  {{/* Limit to the two entries with the closest end dates */}}
+  {{ $entriesToShow := first 2 $sortedAndFiltered }}
 
 <div class="o-banner">
 
-<i class="fas fa-bullhorn"></i> Join us for 
-[**OTel Community Day** on June 25th](https://sessionize.com/OTel-Community-Day/)!
+  {{ range $entriesToShow }}
 
+<i class="fas fa-bullhorn"></i> 
```

Comment:
```suggestion
<i class="fas fa-bullhorn"></i> 
```

Can we make that bullhorn also configurable? So default to it, if people do not provide another icon from font awesome. We do something similar in the `hugo.yaml` for the icons on the bottom of the page:

```
      - name: Mailing Lists
        url: https://github.com/open-telemetry/community#mailing-lists
        icon: fa fa-envelope
        desc: List of mailing lists that the project uses.
      - name: Mastodon
        url: https://fosstodon.org/@opentelemetry
        icon: fab fa-mastodon
        desc: Follow us on Mastodon to get the latest news!
```

Reply:
for sure we can!